annotation-file-utilities/src/annotator/find/ASTPathCriterion.java

```diff
@@ -383,9 +383,8 @@ private Tree getNext(Tree actualNode, ASTPath astPath, int ix) {
                 } else if (astNode.childSelectorIs(ASTPath.BOUND)) {
                   return arg < 0 ? clazz.getExtendsClause()
                           : clazz.getImplementsClause().get(arg);
-                } else {
-                    return null;
                 }
+                return null;
```

Comment:
I made the change but noticed that the compiler gives a warning if the return statement is between lines 382 and 383:
```
     [java] .../annotation-tools/annotation-file-utilities/src/annotator/find/ASTPathCriterion.java:389: warning: [fallthrough] possible fall-through into case
     [java]             case CONDITIONAL_EXPRESSION: {
     [java]             ^
     [java] error: warnings found and -Werror specified
     [java] 1 error
     [java] 1 warning
```
Should I revert them?